spec/unit/hanami/logger_spec.rb

```diff
@@ -672,9 +700,10 @@ class TestLogger < Hanami::Logger; end
     end
 
     context "when input has Tempfile" do
-      it "filters with out errors with closed stream" do
-        tempfile = Tempfile.new("filter_test")
-        tempfile.close!
+      let(:tempfile) { Tempfile.new("filter_test") }
+      after { tempfile.close! }
```

Comment:
@khaile this line changes the semantic of the test. The test tests the case when file is closed before running this log method, while your change make it close after the log method.

Reply:
@hieuk09 thank you for clarifying. I will update my change since this line against the purpose of this PR https://github.com/hanami/utils/pull/342/files I haven't seen before 🙇‍♂️